### PR TITLE
Added stress test with 100 endpoints.

### PR DESCRIPTION
**Pull request by karok2m**
Added stress test with 100 endpoints.

### CODE REVIEW

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs [hunk 1]
```diff
@@ -267,5 +270,139 @@ public class LegacyPublisherConfigServicesTests {
...
+                    .Should()
+                    .Be("Succeeded");
+            }
+
+            async Task CheckEndpointsAndNodes(
+                int expectedNumberOfEndpoints,
+                int expectedNumberOfNodes
+            ) {
+                var tasks = new List<Task<JobProcessingInstructionModel>>();
+                for (var i = 0; i < expectedNumberOfEndpoints; i++) {
```
Comment (cristipogacean) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs]: you can go expectedNumberOfEndpoints+1 to make sure no extra job was created.

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs [hunk 2]
```diff
@@ -267,5 +270,139 @@ public class LegacyPublisherConfigServicesTests {
...
+        [Fact]
+        public async Task PublishNodesStressTest() {
+            var legacyCliModelProviderMock = new Mock<ILegacyCliModelProvider>();
+            var agentConfigProviderMock = new Mock<IAgentConfigProvider>();
+            var identityMock = new Mock<IIdentity>();
+            var newtonSoftJsonSerializer = new NewtonSoftJsonSerializer();
+            var jobSerializer = new PublisherJobSerializer(newtonSoftJsonSerializer);
+            var publishedNodesJobConverter = new PublishedNodesJobConverter(TraceLogger.Create(), newtonSoftJsonSerializer);
+
+            string tempPublishedNodesFile = Path.GetTempFileName();
```
Comment (cristipogacean) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs]: we should probably delete this file as soon as the orchestrator is created and makes use of it. if we delete it only in the end of the testcase we might encounter exceptions and end-up with a bunch of undeleted empty files. 
Alternatively put the entire code in a try...finally block for cleanup.

**Review by cristipogacean — APPROVED** [review]

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs (hunk 1 above)
Comment (karok2m) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs]: Good point. Done.

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs (hunk 2 above)
Comment (karok2m) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Engine/LegacyPublisherConfigServicesTests.cs]: Added removal of temporary file on test cleanup.

**Pull request merged**